gslib/tests/test_gsutil.py

```diff
@@ -18,14 +18,21 @@
 from __future__ import print_function
 from __future__ import division
 from __future__ import unicode_literals
+from ast import Is
```

Comment:
What was Is imported for here?

Reply:
Oops. Not sure how this got added. Removed